### ISSUE TITLE
If it is enabled in the config, let user control the parameters of the receiver hardware 

### ISSUE THREAD

**Issue by ha7ilm** [simonyiszk/openwebrx#37]
These parameters would include:
* Center frequency
* Gain
* Auto gain
* etc...

This was useful for private receivers.

**Comment by LDighera** [comment]
I am pleased to see this enhancement being addressed now.

Thank you for sharing all your hard work.  Very much appreciated.

**Comment by kc9hzc** [comment]
I too am pleased!  These would be welcome options for a private receiver such as mine.  Frequency changes from within the browser session would be at the top of my wish list.  I saw mention of the ability to include '#freq=123456789,mod=fm' within the URL but this does not seem to work for me now.

I love this project, thanks for all of your hard work!!

**Comment by silviuvulcan** [comment]
Hi. Any chance this is still a thing? I like the idea of a private receiver I can access remotely via vpn...

**Comment by ha7ilm** [comment]
Yes, it's not forgotten.

**Comment by Giorgiofox** [comment]
I agree, very very useful to set center frequency from web client for personal server! Please add it as soon as possible, and many thanks for this work!
